qa/workunits/rgw/s3_utilities.pm

```diff
@@ -134,15 +134,31 @@ sub purge_data
     return 0;
 }
 
+# Read PRETTY_NAME from /etc/os-release
+sub os_pretty_name
+{
+        open(FH, '<', '/etc/os-release') or die $!;
+        while (my $line = <FH>) {
+                if($line =~ /^PRETTY_NAME=(.*)$/)
+                {
+                        my $pretty_name = $1;
+                        $pretty_name =~ s/^"//;
+                        $pretty_name =~ s/"$//;
+                        return $pretty_name;
+                }
```

Comment:
```suggestion
                if ($line =~ /^\s*PRETTY_NAME=\"?(.*)\"/) {
                    return $1;
                }
```

Reply:
with this change, the method returns `Fedora 34 (Thirty Four)"` on my system... still investigating...

Reply:
Oh, yeah that `.*` glob is greedy, so it needs to exclude `"` characters.